code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -260,8 +260,6 @@
 		var/position = vending_names_paths.Find(build_path)
 		position = (position == vending_names_paths.len) ? 1 : (position + 1)
 		var/typepath = vending_names_paths[position]
-
-		to_chat(user, "<span class='notice'>You set the board to \"[vending_names_paths[typepath]]\".</span>")
```

Comment:
?

Reply:
probably some redundant text shit idk, assuming you _should_ know what you're setting the board to since you're setting it